packages/orbit-components/src/Slider/components/Handle/index.tsx

```diff
@@ -96,9 +96,9 @@ StyledHandle.defaultProps = {
 
 interface Props {
   tabIndex: string | number;
-  onMouseDown: React.MouseEventHandler<HTMLDivElement>;
```

Comment:
Remove also the `index.js.flow` files from child components, they are not needed, because we do not export them